providers/apis/defi/initia/client.go

```
@@ -0,0 +1,176 @@
+		go func(index int, client Client) {
+			defer wg.Done()
+			resp, err := client.SpotPrice(ctx, denom)
+			if err != nil {
+				mc.logger.Error("failed to spot price in sub client", zap.String("url", url), zap.Error(err))
+				return
+			}
+
+			mc.logger.Debug("successfully fetched accounts", zap.String("url", url))
+
+			resps[index] = resp
+
```

golangci-lint (1.64.8)

File is not properly formatted

(gofumpt)